cfme/tests/test_login.py

```diff
@@ -71,13 +100,13 @@ def test_update_password(context, request, appliance):
 
     # First, create a temporary new user
     username = 'user_temp_{}'.format(fauxfactory.gen_alphanumeric(4).lower())
-    new_creds = Credential(principal=username, secret='redhat')
-    user_group = appliance.collections.groups.instantiate(description="EvmGroup-vm_user")
-    user = appliance.collections.users.create(
-        name=username,
-        credential=new_creds,
-        groups=user_group
+    user = new_user_create(
```

Comment:
This is changing test behavior by not passing secret, which is explicitly set in the old test. Reasoning for the change?